src/scheduler/job_info.c

```diff
@@ -521,6 +521,8 @@ query_jobs(status *policy, int pbs_sd, queue_info *qinfo, resource_resv **pjobs,
 	struct attropl opl = { NULL, ATTR_q, NULL, NULL, EQ };
 	static struct attropl opl2[2] = { { &opl2[1], ATTR_state, NULL, "Q", EQ},
 		{ NULL, ATTR_array, NULL, "True", NE} };
+	struct attrl *attrib = NULL;
```

Comment:
Why don't we make this a static list and reuse it every time scheduler queries jobs?